src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -184,6 +184,8 @@ private void SetGlobalErrorIfTrue(bool arg)
             }
         }
 
+        // Returns the MethodSymbol for the assembly entrypoint.  If the user has a Task returning main,
+        // this function returns the synthesized Main MethodSymbol.
```

Comment:
If we have to specify two entry points - actual and for debugging purposes, it would be a different change, I guess.

---
In reply to: [111274974](https://github.com/dotnet/roslyn/pull/18472#discussion_r111274974) [](ancestors = 111274974)